website/docs/Queries.md

```diff
@@ -359,57 +358,31 @@ render(<Component />);
 const element = screen.getByA11yValue({ min: 40 });
 ```
 
-## Hidden accessibility option
+## Common query options
 
-All queries have the `hidden` option which enables them to respect accessibility props on components when it is set to `false`. If you set `hidden` to `true`, elements that are normally excluded from the accessibility tree are considered for the query as well.  For now if no `hidden` option is defined, `hidden` is set to `true` by default in order to avoid breaking changes. In the near future, we plan to set it to `false` by default so that the default behavior of all queries is to respect accessibility.
+### `hidden` option
 
-An element will be considered inaccessible when it checks one of the following: 
-- it has a style with `display: "none"`
-- it has the prop [`importantForAccessibility='no-hide-descendants'`](https://reactnative.dev/docs/accessibility#importantforaccessibility-android)
-- it has the prop [`accessibilityElementsHidden={true}`](https://reactnative.dev/docs/accessibility#accessibilityelementshidden-ios) 
-- one of its siblings has the prop [`accessibilityViewIsModal={true}`](https://reactnative.dev/docs/accessibility#accessibilityviewismodal-ios)
-- one of its parents is inaccessible due to one of the checks above
+All queries have the `hidden` option which enables them to respect accessibility props on components when it is set to `false`. If you set `hidden` to `true`, elements that are normally excluded from the accessibility tree are considered for the query as well.  Currently `hidden` option is set `true` by default, which means that elements hidden from accessibility will be included by default. However, we plan to change the default value to `hidden: false` in the next major release.
 
-You can find more documentation about accessibility in React Native [here](https://reactnative.dev/docs/accessibility).
+You can configure the default value with the [`configure` function](API.md#configure).
 
-### Examples
-
-#### `display: "none"`
-Given the following render: 
-```ts
-render(<Text style={{display:"none"}}>I am inaccessible</Text>)
-```
+An element is considered to be hidden from accessibility based on [`isInaccessible()`](./API.md#isinaccessible) function result.
 
-Will **find a match**:
-```ts
-screen.getByText("I am inaccessible", { hidden: true});
-screen.getByText("I am inaccessible");
-```
+#### Examples
 
-Will **NOT find a match**:
-```ts
-screen.getByText("I am inaccessible", { hidden: false }); 
-```
-
-#### `accessibilityElementsHidden`
+Given the following render: 
```

Comment:
This looks like a left over from previous version of the doc
```suggestion
```

Reply:
looks like a git glitch, I have already deleted it